### PR TITLE
Update Set-RetentionCompliancePolicy.md

### PR DESCRIPTION
**Pull request by scorrea4099**
Document update request to add a simple note recommending against using piped Foreach-Object when adding or removing scope Locations, as its very inefficient vs the recommended comma delimited values, it can potentially cause possible policy scope corruption, which can result in Admins being unable to pull the policy when the DistributionDetail parameter is utilized, a requirement for accurate status :  `Get-RetentionCompliancePolicy <Identity> -DistributionDetail`, the full root cause is still pending and it primarily impacts RetentionCompliancePolicy's.

Please let me know if this would be better served as a single item note at the top somewhere for visibility instead on every scope Location related parameter
Also, if for consistency we would need to do it to all 5 set policy cmdlets where comma delimited scope Locations are mentioned

https://docs.microsoft.com/powershell/module/exchange/set-caseholdpolicy
https://docs.microsoft.com/powershell/module/exchange/set-dlpcompliancepolicy
https://docs.microsoft.com/powershell/module/exchange/set-holdcompliancepolicy
https://docs.microsoft.com/powershell/module/exchange/set-labelpolicy
https://docs.microsoft.com/powershell/module/exchange/set-retentioncompliancepolicy - This change

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Review by dariomws — COMMENTED** [review]
@scorrea4099, I don't have permission to see the work item but if it was reported for some customer, I think we can add a single note in the description section of all the related cmdlets.

**Comment by scorrea4099** [comment]
Moved the Note to the Synopsis description, removed from the scope Location parameters

**Review by tseward — APPROVED** [review]

File: exchange/exchange-ps/exchange/Set-RetentionCompliancePolicy.md [hunk 1]
```diff
@@ -88,6 +88,8 @@ Set-RetentionCompliancePolicy [-Identity] <PolicyIdParameter>
 ## DESCRIPTION
 To use this cmdlet in Security & Compliance Center PowerShell, you need to be assigned permissions. For more information, see [Permissions in the Microsoft 365 compliance center](https://docs.microsoft.com/microsoft-365/compliance/microsoft-365-compliance-center-permissions).
 
+**Note**: Please avoid using a piped Foreach-Object when adding or removing scope Locations : "Value1","Value2",..."ValueN" | Foreach-Object { Set-RetentionCompliancePolicy -Identity "Regulation 123 Compliance" -RemoveExchangeLocation $_  }
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-RetentionCompliancePolicy.md:91]:
```suggestion
**Note**: Avoid using a piped Foreach-Object when adding or removing scope Locations : "Value1","Value2",..."ValueN" | Foreach-Object { Set-RetentionCompliancePolicy -Identity "Regulation 123 Compliance" -RemoveExchangeLocation $_  }
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @scorrea4099, kindly commit the suggested change. For information, we do not use "please" except in certain circumstances (which do not apply in this text). Many thanks.

**Review by dariomws — APPROVED** [review]

File: exchange/exchange-ps/exchange/Set-RetentionCompliancePolicy.md (hunk 1 above)
Comment (scorrea4099) [reply exchange/exchange-ps/exchange/Set-RetentionCompliancePolicy.md]: @chrisda  is going with "Don't use a piped Foreach-Object" for the other PRs

**Pull request merged**